Makefile

```diff
@@ -28,13 +29,21 @@ VERSION := ${shell cat ./VERSION}
 
 SHELL := $(shell command -v bash 2>/dev/null)
 
-all: $(RUNC_LINK)
+.DEFAULT: runc
+
+all: runc recvtty
+
+runc: $(SOURCES) | $(RUNC_LINK)
 	go build -i -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION}" -tags "$(BUILDTAGS)" -o runc .
 
-static: $(RUNC_LINK)
+recvtty: $(SOURCES) | $(RUNC_LINK)
```

Comment:
Since you are adding an extra binary build target, we need to add another copy to the dbuild target: 
```
diff --git a/Makefile b/Makefile
index 29786f8..1cbdd58 100644
--- a/Makefile
+++ b/Makefile
@@ -12,6 +12,7 @@ PROJECT := github.com/opencontainers/runc
 TEST_DOCKERFILE := script/test_Dockerfile
 BUILDTAGS := seccomp
 RUNC_BUILD_PATH := /go/src/github.com/opencontainers/runc/runc
+RECVTTY_BUILD_PATH := /go/src/github.com/opencontainers/runc/recvtty
 RUNC_INSTANCE := runc_dev
 COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
 COMMIT := $(if $(shell git status --porcelain --untracked-files=no),"${COMMIT_NO}-dirty","${COMMIT_NO}")
@@ -75,6 +76,7 @@ $(RUNC_LINK):
 dbuild: runcimage
        docker run --name=$(RUNC_INSTANCE) --privileged $(RUNC_IMAGE) make
        docker cp $(RUNC_INSTANCE):$(RUNC_BUILD_PATH) .
+       docker cp $(RUNC_INSTANCE):$(RECVTTY_BUILD_PATH) .
        docker rm $(RUNC_INSTANCE)
 
 lint:

```